src/main/java/Duke.java

```diff
@@ -1,10 +1,99 @@
+import java.util.Locale;
+import java.util.Scanner;
+
 public class Duke {
+
+    /* User input is seperated by an empty space */
+    public static final String USER_INPUT_SEPERATOR = " ";
+    /* User command list */
+    public static final String USER_COMMAND_LIST = "LIST";
+    public static final String USER_COMMAND_TODO = "TODO";
+    public static final String USER_COMMAND_DEADLINE = "DEADLINE";
+    public static final String USER_COMMAND_EVENT = "EVENT";
+    public static final String USER_COMMAND_DONE = "DONE";
+    public static final String USER_COMMAND_BYE = "BYE";
+    public static final String USER_COMMAND_HELP = "HELP";
+
+    /* A nicely formatted line */
+    private static final String LINE = "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=";
+    private static final String HELP_MESSAGE = "The following commands accepted are: "
+            + "LIST (Show the list of task)\n"
+            + "TODO <description> (Create a task with todo tag)\n"
+            + "DEADLINE <description> /by <date and time> (Create a task with deadline tag)\n"
+            + "EVENT <description> /at <date and time> (Create a task with event tag)\n"
+            + "DONE <index of task> (To mark indicated task as completed)\n"
+            + "BYE (End program)\n"
+            + "HELP (List out available commands)";
+
+
+    /**
+     * Prints given string in the middle of 2 horizontal lines.
+     *
+     * @param message String to be printed
+     */
+    public static void printMessage(String message) {
+        System.out.println(LINE);
+        System.out.println(message);
+        System.out.println(LINE);
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        // Set welcome and exit message for user
+        String welcomeMessage = "Hello! I'm Gaben.\n"
+                + "Steam sales is coming! Prepare your wallet.\n"
+                + "Anyways, what can I do for you today?";
+        String exitMessage = "Thank you for using Gaben.\n"
+                + "Remember to keep your wallet stacked before using me again!";
+
+        // Print welcome message to user
+        printMessage(welcomeMessage);
+        // Create task manager to manage task given by user
```

Comment:
I would suggest to include a space above this comment (and any other similar lines) to improve readability.